shell/platform/embedder/embedder.cc

```diff
@@ -1434,11 +1509,16 @@ CreateEmbedderRenderTarget(
       break;
     }
     case kFlutterBackingStoreTypeVulkan: {
-      auto skia_surface =
-          MakeSkSurfaceFromBackingStore(context, config, &backing_store.vulkan);
-      render_target = MakeRenderTargetFromSkSurface(
-          backing_store, std::move(skia_surface), collect_callback.Release());
-      break;
+      if (enable_impeller) {
+        FML_LOG(ERROR) << "Unimplemented";
+        break;
```

Comment:
IIRC, this should return nullptr. That should return the right error code to the embedder. We can file a bug for followup.

Reply:
Breaking here leaves the render_target unique_ptr as nullptr which accomplishes the same thing